addons/accessory/XEH_preInit.sqf

```diff
@@ -2,7 +2,7 @@
 
 if (!hasInterface) exitWith {};
 
-#include "XEH_PREP.sqf"
+#include "XEH_PREP.hpp"
```

Comment:
not sure if hpp is right, its not a config, it's still just sqf?

```suggestion
#include "XEH_PREP.inc.sqf"
```

Reply:
I'm fine with either, just used `.hpp` since that's what ACE uses

Reply:
oh right, then it's fine